docs/developer/DEVELOP.md

```diff
@@ -18,6 +18,11 @@ BZIMPORT_BZ_API_KEY=####################
 JIRAFFE_AUTO_SYNC=True
 JIRA_AUTH_TOKEN=####################
 
+# Enable task management in Jira
+JIRA_TASKMAN_URL="https://foo.bar"
+JIRA_TASKMAN_TOKEN=####################
```

Comment:
I'm in favor to do it using user token but I have few concerns that I will be addressing today:
- Using the user token for the creation will require creating a Atlassian app for OSIDB and the user itself authorizing us to create things in Jira in behalf of them, I'm not sure if any authorization is needed from our Jira admins but if it is needed it is a possible blocker
- I'm not sure if our current Jira lib supports impersonation since its a feature added by Atlassian in December, the solution for that would be creating a manual client for Jira (but I need to check, maybe it is already supported)

Reply:
Discussed during the meeting. No special functionality needed. Only the user token will be passed instead of the service one when creating the connection - the connection class (or its child) needs some extension.

Reply:
I have changed the `JiraTaskmanQuerier` in order to not impact Jiraffe module that aways use the service account. Please take a look and see if it address the problem.